exporter/kinesisexporter/README.md

```diff
@@ -1,3 +1,48 @@
 # Kinesis Exporter
 
-To be added.
\ No newline at end of file
+Kinesis exporter exports OpenTelemetry data to Kinesis. This exporter uses a [KPL][kpl-url]-like batch producer and uses
+the same aggregation format that KPLs use. Message payload encoding is configurable.
+
+The following settings can be optionally configured:
+- `aws` contains AWS specific configuration
+  - `stream_name` (default = test-stream): The Kinesis stream to push events to
+  - `kinesis_endpoint`: The Kinesis endpoint if role is not getting assumed
+  - `region` (default = us-west-2): The AWS region where the Kinesis stream is defined
+  - `role`: The Kinesis role to assume
+- `kpl` contains kinesis producer library related config to controls things like aggregation, batching, connections, retries, etc
+  - `aggregate_batch_count` (default = 4294967295): Determines the maximum number of items to pack into an aggregated record. Must not exceed 4294967295
+  - `aggregate_batch_size` (default = 51200): Determines the maximum number of bytes to pack into an aggregated record. User records larger than this will bypass aggregation
```

Comment:
What does "bypass aggregation" mean?

Reply:
The kinesis producer library logical aggregates multiple records into a single kinesis record for efficient puts. If the record size is greater than the aggregate batch size we set up then it just gets treated as a normal kinesis record rather than get aggregated: https://github.com/signalfx/omnition-kinesis-producer/blob/master/producer.go#L93